cmd/provider/main.go

```diff
@@ -37,6 +37,7 @@ func main() {
 		app            = kingpin.New(filepath.Base(os.Args[0]), "AWS support for Crossplane.").DefaultEnvars()
 		debug          = app.Flag("debug", "Run with debug logging.").Short('d').Bool()
 		syncPeriod     = app.Flag("sync", "Controller manager sync period duration such as 300ms, 1.5h or 2h45m").Short('s').Default("1h").Duration()
+		pollInterval   = app.Flag("poll-interval", "Default poll interval for all manage resource controllers.").Default("1m").Duration()
```

Comment:
```suggestion
		syncInterval     = app.Flag("sync", "Controller manager sync interval.").Short('s').Default("1h").Duration()
		pollInterval   = app.Flag("poll", "Default poll interval for all manage resource controllers.").Default("1m").Duration()
```

What do you think about this pattern? This way we're consistent with the existing `--sync` flag that serves a similar purpose. Speaking of which, I could easily see readers having a hard time distinguishing between those two flags. It would be ideal to try to describe what the difference is between the sync period and the poll interval in the help text, though it might be tough to do succinctly.

Reply:
I like it! Let me take a pass at updating the help text here and let you take another look :+1: